.github/workflows/dart.yml

```diff
@@ -106,6 +106,57 @@ jobs:
           OTHER_SWIFT_FLAGS: "-warnings-as-errors"
           DEVELOPER_DIR: /Applications/Xcode_${{ matrix.xcode }}.app/Contents/Developer
 
+  build-ios-spm:
+    name: Build ${{ matrix.package }} iOS on ${{ matrix.os }} with Xcode ${{ matrix.xcode }} and Flutter ${{ matrix.flutter }} with SPM
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-15]
+        package: [sign_in_with_apple/sign_in_with_apple]
+        xcode: ["16.1", "16.4"]
+        flutter: ["3.27.2", "3.32.4"]
+        exclude:
+          - xcode: 16.1
+            flutter: 3.32.4
+          - xcode: 16.4
+            flutter: 3.27.2
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-java@v1
+        with:
+          java-version: "12.x"
+      - name: Xcode select
+        run: sudo xcode-select -s /Applications/Xcode_${{ matrix.xcode }}.app
+      - uses: subosito/flutter-action@4389e6cbc6cb8a4b18c628ff96ff90be0e926aa8 # v1.5.3
+        with:
+          flutter-version: ${{ matrix.flutter }}
+      - name: Flutter doctor (version check)
+        run: flutter doctor
+      - name: Enable SPM support
+        run:  flutter config --enable-swift-package-manager
+        working-directory: packages/${{ matrix.package }}/example
+      - name: List dependency config before (debug)
+        run: |
+          less ios/Runner.xcodeproj/project.pbxproj | grep Pods || echo ""
+          less ios/Runner.xcodeproj/project.pbxproj | grep FlutterGeneratedPluginSwiftPackage || echo ""
+        working-directory: packages/${{ matrix.package }}/example
+      - name: Remove Cocoapods from project
```

Comment:
I guess also other Flutter projects still support cocoapods or how is this looking there?

Reply:
Yeah, either way I would like to test both for now.

And I think since SPM is not yet the default one, it makes sense to keep the CP project and just migrate it in CI.

But if need be we can of course add a second clean SPM one (hopefully this can be avoided and we just switch to all-on SPM soon enough).